### DISCUSSION TITLE
Redis dosent contain updated leafnodes

### DISCUSSION THREAD

**Discussion by somvitbhowmik** [Sunbird-Knowlg/Community#70] [General Discussions]
Hi Team,

Redis dosent not contain latest leafnodes as we have in hierarchy. We have tried to update the redis leafnodes using jenkins job. But it is not updated with the latest data. KP Flink job version is release 4.3.0. Let me know how to solve this data. @maheshkumargangula @Jayaprakash8887.

For ex: smembers "do_113502156371935232194:do_113502156371935232194:leafnodes"

cc: @pkranga

Regards,
Somvit

**Comment by Jayaprakash8887** [comment]
@somvitbhowmik https://github.com/project-sunbird/knowledge-platform-jobs/blob/release-4.3.0/relation-cache-updater/src/main/scala/org/sunbird/job/relationcache/functions/RelationCacheUpdater.scala

line 65 'storeDataInCache' method

RelationCacheUpdater job should be performing this activity as per https://github.com/project-sunbird/knowledge-platform-jobs/blob/release-4.3.0/relation-cache-updater/README.md

**Reply by Jayaprakash8887** [reply]
Job is owned by Sunbird LERN team. Kindly check with the LERN team.
